### ISSUE TITLE
DmSortable alike interface for NestedSets in admin

### ISSUE THREAD

**Issue by 4levels** [diem-project/diem#269]
Hi Diem Team,

Diem provides a very nice jQuery drag'n'drop feature in the admin to sort records having the DmSortable behaviour.  Can this also be achieved for records behaving as NestedSets?  Instead of showing a flat list of draggable buttons as the DmSortable interface, the same jQuery tree as seen on the seo/pages module could be used instead.  As the seo/pages already features a very clever drag'n'drop tree interface, this looks really possible to implement this for any nestedSet module.

To me a nestedSet is also a sortable behaviour, actually an even more advanced sorting, hence I feel like this is really lacking in the admin.  My clients really don't understand the root_id, lft, rgt, level columns ...

To summarize: as soon as an admin module has a NestedSet model, a link 'Sort' is provided taking the user to a tree view where the records can be drag'n'dropped around the tree.
